### PR TITLE
refactor(navbar): improve navbar samples

### PR DESCRIPTION
**Pull request by SisIvanova**
Closes: #1843

### CODE REVIEW

File: src/app/menus/navbar/navbar-style/navbar-style.component.scss [hunk 1]
```diff
@@ -37,23 +37,23 @@ $dark-navbar: igx-navbar-theme(
...
+//   $palette: $light-navbar-palette,
+//   $schema: $custom-light-schema
 // );
 
 /*INCLUSION by DEFAULT ViewEncapsulation strategy*/
 
 :host {
      ::ng-deep {
-        @include igx-navbar($dark-navbar);
+        @include igx-navbar($custom-navbar-theme);
```
Comment (simeonoff) [inline src/app/menus/navbar/navbar-style/navbar-style.component.scss]: Shouldn't this sample include the `igx-css-vars` mixin instead?

File: src/app/menus/navbar/navbar-sample-3/navbar-sample-3.component.html [hunk 2]
```diff
@@ -0,0 +1,6 @@
+<article class="sample-column">
```
Comment (simeonoff) [inline src/app/menus/navbar/navbar-sample-3/navbar-sample-3.component.html]: Do we need this extra element?

File: src/app/menus/navbar/navbar.component.html [hunk 3]
```diff
@@ -1,30 +1,6 @@
...
-                <h4 class="sample-title">Without nav icon</h4>
-                <div class="navbar-sample">
-                    <igx-navbar [title]="currentView" [isActionButtonVisible]="false">
-                    </igx-navbar>
-                </div>
-            </article>
-        </section>
-    </div>
\ No newline at end of file
+<article class="sample-column">
```
Comment (simeonoff) [inline src/app/menus/navbar/navbar.component.html]: Same here.

Comment (SisIvanova) [reply src/app/menus/navbar/navbar.component.html]: This class brings the margin of the navbar so I will leave it if you don't mind. However, I changed it to div and removed the following one.

**Pull request merged**